packages/platform-sdk-generator/src/model/Model.ts

```diff
@@ -95,9 +100,17 @@ export class Model {
     }
   }
 
-  getComponent(reference: string): Component {
-    const ret = this.#components.get(reference);
-    if (!ret) throw new Error(`Component ${reference} not found`);
+  getComponent(locator: ir.Locator): Component {
+    const ret = this.#components.get(
+      `${
+        locator.namespaceName === "Ontologies" ? "Core" : locator.namespaceName
```

Comment:
Discussed on slack, but BLUF is we will continue with the "hack". Because we don't use any v1 api's in out code, we don't want to depend on that package. This hack allows us to export types used in many packages outside of the `internal.foundry.ontologies` package, which are not present in the v2 version. Potential FLUPs might include not generating this package at all, or moving the types into `internal.foundry.ontologiesv2`